state-chain/chains/Cargo.toml

```diff
@@ -50,7 +50,7 @@ generic-array = { version = "0.14.7", default-features = false, features = [
   "serde",
   "more_lengths",
 ] }
-bs58 = { version = "0.4.0", default-features = false, features = ["alloc"] }
+bs58 = { version = "^0.5", default-features = false, features = ["alloc"] }
```

Comment:
I'd advocate for using exact versions. For another PR in the engine we will use `0.5.1` so I'd use that one

Reply:
Also FYI the "^" is unnecessary. `"^0.x.y"` is the same as `"0.x.y"`

Reply:
Updated to 0.5.1